File: library_arouter/src/main/java/cn/jack/library_arouter/router/RouterPathActivity.java
package cn.jack.library_arouter.router;

/**
 * created by Jack
 * email:yucrun@163.com
 * date:19-4-16
 * describe:
 */
public class RouterPathActivity {

    public static class Home {
        private static final String HOME = "/home";
        public static final String PAGER_HOME = HOME + "/Home";
    }

    public static class Login {
        private static final String LOGIN = "/login";
        public static final String PAGER_LOGIN = LOGIN + "/Login";
    }

    public static class Register {
        private static final String REGISTER = "/register";
        public static final String PAGER_REGISTER = REGISTER + "/Register";
    }

    public static class Subject {
        private static final String SUBJECT = "/subject";
        public static final String PAGER_SUBJECT = SUBJECT + "/Subject";
    }
}

File: library_common_arouter/src/main/java/cn/jack/library_arouter/router/RouterPathActivity.kt
package cn.jack.library_arouter.router

/**
 * created by Jack
 * date:19-4-16
 * describe:
 */
class RouterPathActivity {
    object Home {
        private const val HOME = "/home"
        const val PAGER_HOME = "$HOME/Home"
    }

    object Login {
        private const val LOGIN = "/login"
        const val PAGER_LOGIN = "$LOGIN/Login"
    }

    object Register {
        private const val REGISTER = "/register"
        const val PAGER_REGISTER = "$REGISTER/Register"
    }

    object Subject {
        private const val SUBJECT = "/subject"
        const val PAGER_SUBJECT = "$SUBJECT/Subject"
    }

    object Web {
        private const val WEB = "/web"
        const val PAGER_WEB = "$WEB/Web"
    }
}